translations/nl-be.json

```diff
@@ -813,5 +813,7 @@
   "submission-activity": "Indieningsactiviteit",
   "config": "Instelling",
   "type": "Type",
-  "archive": "Archief"
+  "archive": "Archief",
+  "not-entered-yet": "Nog niet ingevuld",
+  "withdrawn-on": "Ingetrokken op"
```

Comment:
"Ingetrokken" is niet de correcte term bij statussen van de publication-flow, de juiste term is "Afgevoerd".
Is verwarrend omdat "withdrawn" nog op sommige plaatsen gebruikt wordt, een alternatieve engelse vertaling is "discharged".

Reply:
Begrijpelijke verwarring, gezien de figma-file de oude terminologie gebruikt. https://www.figma.com/file/R08Q4OJ1Nmm0I7uCw2DGUW/%F0%9F%92%AF-Kaleidos---Designs-Digitale-Publicaties-v2?node-id=2366%3A0 We willen inderedaad "afgevoerd op"